embulk-input-s3/src/test/java/org/embulk/input/s3/TestS3InputStreamReopener.java

```diff
@@ -89,6 +90,28 @@ public void reopenS3FileByReopener_on_retry_gave_up_should_throw_original_except
         opener.reopen(0, new RuntimeException());
     }
 
+    @Test(expected = AmazonClientException.class)
+    public void reopenS3FileByReopener_on_retry_always_throw_exception()
```

Comment:
I assume this test intends to verify when the retry attempts exceed the max number, `reopen()` will throw? If so, this method is kind of misleading and actually throws an alarm instead, please consider another name, and how is this different with the exists test `reopenS3FileByReopener_on_retry_gave_up_should_throw_original_exception`?